auth.ts

```diff
@@ -153,17 +153,24 @@ interface AuthInit {
   private_key: string;
   private_key_id: string;
   project_id: string;
+  datastore_host?: string;
```

Comment:
Why not?

```suggestion
  host?: string;
```

Reply:
the env var generated by the datastore emulator that would be passed is called `DATASTORE_HOST`